src/errors/BindingStrengthError.ts

```diff
@@ -2,7 +2,10 @@ import { Annotated } from './Annotated';
 
 export class BindingStrengthError extends Error implements Annotated {
   specReferences = ['http://hl7.org/fhir/R4/terminologies.html#strength'];
-  constructor(public foundStrength: string, public requestedStrength: string) {
```

Comment:
Prettier is supposed to be the one with the opinions, but I just think the original format was prettier than this. But oh well.

Reply:
It seems like it only made this change for constructors with `public`/`private` modifiers, so I suspect the intent of prettier here is to make these arguments stand out more since they're also a class member declaration.

Reply:
Ah, perhaps.